generator/source/index.md

```diff
@@ -1,24 +1,36 @@
 ---
 hide: true
 layout: page
+title: Hyperledger Grid
 permalink: /
 feature-img: "images/hero-bg.jpg"
 # Copyright (c) 2018 Bitwise IO, Inc.
 # Licensed under Creative Commons Attribution 4.0 International License
 # https://creativecommons.org/licenses/by/4.0/
 ---
 
-# Hyperledger Grid
+Hyperledger Grid is a platform for building supply chain solutions that include
+distributed ledger components. It provides a set of libraries, data models, and
+SDKs for developing supply chain smart contracts and client interfaces.
 
-Hyperledger Grid is a project for building supply chain solutions. It includes
-a set of libraries, data models, and SDKs to accelerate development for supply
-chain smart contracts and client interfaces.
+This project's goal is to accelerate development of modular blockchain-based
+solutions for cross-industry supply chain problems. Hyperledger Grid provides
+domain-specific data models, such as GS1 product definitions that can be used in
+a product catalog, example smart contracts, and other modular components.
```

Comment:
This feels redundant and I'd just combine it with the above.

Reply:
@davececchi - I reworked the first paragraph to include the non-duplicate content from the second paragraph. Is the new version any better?